scripts/validate_readme_structure.py

```
@@ -0,0 +1,741 @@
+    base_dir: Path,
+) -> ValidationResult:
+    """Validate a README against the template structure."""
+    result = ValidationResult(
+        readme_path=readme_path,
+        plugin_type=plugin_type,
+        plugin_name=plugin_name,
+    )
+    
+    try:
+        content = readme_path.read_text()
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)